### PR TITLE
[Validator] Updated Greek translation

### PR DESCRIPTION
**Pull request by thunderer**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | #51942
| License       | MIT

Missing Greek translations with help from a native speaker.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
...
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Αυτή η τιμή περιέχει χαρακτήρες που δεν επιτρέπονται από το τρέχον επίπεδο περιορισμού.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>Δεν επιτρέπεται η χρήση αόρατων χαρακτήρων.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>Δεν επιτρέπεται η μίξη αριθμών από διαφορετικά σενάρια.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: ?
```suggestion
                <target>Δεν επιτρέπεται η μίξη αριθμών από διαφορετικά γραφήματα.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf [hunk 2]
```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Η τιμή του netmask πρέπει να είναι ανάμεσα σε {{ min }} και {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Το όνομα αρχείου είναι πολύ μεγάλο. Θα πρέπει να έχει έως {{ filename_max_length }} χαρακτήρες.|Το όνομα αρχείου είναι πολύ μεγάλο. Θα πρέπει να έχει έως {{ filename_max_length }} χαρακτήρες.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]:
```suggestion
                <target>Το όνομα αρχείου είναι πολύ μεγάλο. Θα πρέπει να έχει έως {{ filename_max_length }} χαρακτήρα.|Το όνομα αρχείου είναι πολύ μεγάλο. Θα πρέπει να έχει έως {{ filename_max_length }} χαρακτήρες.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf (hunk 1 above)
Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: @nicolas-grekas applied your suggestion.

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf (hunk 2 above)
Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: @nicolas-grekas applied your suggestion.

**Comment by nicolas-grekas** [comment]
Thank you @thunderer.

**Pull request merged**